### PR TITLE
Remove openshift.common.{is_atomic|is_containerized}

### PR DESCRIPTION
**Pull request by michaelgugino**
We set these variables using facts in init, no need
to duplicate the logic all around the codebase.

### CODE REVIEW

File: playbooks/common/openshift-cluster/upgrades/upgrade_nodes.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@
 
   - name: Drain Node for Kubelet upgrade
     command: >
-      {{ hostvars[groups.oo_first_master.0].openshift.common.client_binary }} adm drain {{ openshift.node.nodename | lower }} --config={{ openshift.common.config_base }}/master/admin.kubeconfig --force --delete-local-data --ignore-daemonsets
+      {{ hostvars[groups.oo_first_master.0]['first_master_client_binary'] }} adm drain {{ openshift.node.nodename | lower }} --config={{ openshift.common.config_base }}/master/admin.kubeconfig --force --delete-local-data --ignore-daemonsets
```
Comment (kwoodson) [inline playbooks/common/openshift-cluster/upgrades/upgrade_nodes.yml]: Is `first_master_client_binary` correct?  Wasn't sure where this was getting set.

On a side note, any plans for the other two variables?
`openshift.node.nodename`
`openshift.common.config_base`

If not, just wanted to put them on the radar.

Comment (michaelgugino) [reply playbooks/common/openshift-cluster/upgrades/upgrade_nodes.yml]: first_master_client_binary is kind of a special case unfortunately.  When you use delegate to, the variables assigned will be from the play host, not the host the task actually runs on.

Unfortunately, even though the role that provides the default is in scope, those role defaults don't make it into a host's host_vars during that play, so there's no way to access them with delegate_to.  In the iniit/facts.yml, I added a small play that runs on the first master to provide this set_fact that can be used here and various other places.

Comment (michaelgugino) [reply playbooks/common/openshift-cluster/upgrades/upgrade_nodes.yml]: I think nodename will be tricky as it's consumed by other methods in openshift_facts, but config_base should be simple enough.  I'm trying to keep these patches somewhat small, so somethings might not be addressed.

**Pull request merged**